### DISCUSSION TITLE
Is there a way to specify content ordering with a file instead of numeric prefixes

### DISCUSSION THREAD

**Discussion by remiconnesson** [nuxt/content#2257] [Q&A]
I need to insert a page in position 2 let's say, but because of that I have to `git mv` everything that comes after position 2.

Is there a way to get automatic ordering from a file for example: `ordering.yml`

```
index.md
get_started/
onboarding/
license.md
api_ref/
```

**Comment by nobkd** [comment]
Not sure if this helps, as it is not anywhere near to your preferred solution, but you could just leave empty areas between the number prefixes:

```
1.index.md
5.get-started/
10.onboarding/
100.license.md
```

And later you could add `7.about.md` just in between...

**Reply by remiconnesson** [reply]
Thank you, it is helpful. I would rather have a file, because it creates a messy file structure but it can do in a pinch.
